tests/test_config.py

```diff
@@ -0,0 +1,23 @@
+
+import aflr
+from aflr import Script, Sound, Speech, Mastering, Voice
+import pprint
+import pytest
+
+aflr.api_key = ""
```

Comment:
We need some environment variables here. @tonythree thoughts on how we should do this?